plugin-dev/Sentry.uplugin

```diff
@@ -21,5 +21,17 @@
 			"LoadingPhase": "Default",
 			"WhitelistPlatforms": [ "Win64", "Mac", "Android", "IOS", "Linux" ]
 		}
-	]
+	],
+	"PostBuildSteps":
+	{
+		"Mac": [
+			"sh $(PluginDir)/Scripts/upload-debug-symbols.sh $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
+		],
+		"Linux": [
+			"sh $(PluginDir)/Scripts/upload-debug-symbols.sh $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
+		],
+		"Win64": [
+			"powershell -ExecutionPolicy RemoteSigned -File $(PluginDir)/Scripts/upload-debug-symbols-win.ps1 $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
+		]
+	}
 }
```

Comment:
Can we skip the step of merging these binaries to the repo, thus increasing `.git` directory size forever and instead add the download to CLI https://github.com/getsentry/sentry-unity/blob/938bb4787911171c443b499642e76c9780b13e3c/scripts/download-sentry-cli.ps1

Reply:
So instead of keeping Sentry CLI executable in the repo we'll be downloading it every time PostBuildSteps are triggered, right?

Reply:
No, we download it in ci and package in the release. Same as other ThirdParty libs

Reply:
search for that script name in the sentry-unity repo to see how it's used in CI

Reply:
Oh, I've misunderstood the original question. Yes, that totally makes sense